TTHAnalysis/python/plotter/wzsm/cuts_www.txt

```diff
@@ -0,0 +1,12 @@
+alwaystrue: 1
```

Comment:
This is a duplicate of cuts_wwwsm.txt that I was studying some time ago, so this should go into that file

Reply:
Just removed it. It was a short-lived test anyway